packages/components/src/components/shell-panel/shell-panel.tsx

```diff
@@ -98,14 +98,14 @@ export class ShellPanel extends LitElement {
   /**
    * When `layout` is `horizontal`, specifies the maximum height of the component.
    *
-   * @deprecated Use the `height` property instead.
+   * @deprecated in v3.0, removal target v6.0 - Use the `height` property instead.
    */
   @property({ reflect: true }) heightScale: Scale;
 
   /**
    * The direction of the component.
    *
-   * @deprecated No longer necessary.
+   * @deprecated in v4.0, removal target v6.0 -  No longer necessary.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated in v4.0.0, removal target v6.0.0 -  No longer necessary.
```